components/dashboard/src/Menu.tsx

```diff
@@ -200,11 +199,6 @@ export default function Menu() {
         }
         // User menu
         return [
-            {
-                title: "Workspaces",
-                link: "/workspaces",
-                alternatives: ["/"],
```

Comment:
@gtsiolis, it the `/` alternative a feature to redirect from `gitpod.io/` to `gitpod.io/workspaces`?

Reply:
@AlexTugarev Ah, yes it is. We should probably move this below, right?